hlx_statics/scripts/lib-adobeio.js

```diff
@@ -194,32 +194,13 @@ export function decorateInlineCodes(element) {
  * @param {*} container The container to inspect
  */
 export function buildCodes(container) {
-  const codes = [...container.querySelectorAll('main > div pre')];
```

Comment:
temporarily reverting this to make RedoclyAPIBlock, but will re-add to the codebase for https://jira.corp.adobe.com/browse/DEVSITE-1454